pkg/clients/elasticache/elasticache.go

```diff
@@ -64,7 +64,7 @@ func NewCreateReplicationGroupInput(g v1beta1.ReplicationGroupParameters, id str
 		AutomaticFailoverEnabled:   g.AutomaticFailoverEnabled,
 		CacheParameterGroupName:    g.CacheParameterGroupName,
 		CacheSecurityGroupNames:    g.CacheSecurityGroupNames,
-		CacheSubnetGroupName:       g.CacheSubnetGroupName,
+		CacheSubnetGroupName:       aws.String(g.CacheSubnetGroupName),
```

Comment:
This causes a unit test failure, but I believe it will not longer be an issue if we go back to using `*string` here.

```
[2020-08-05T01:42:19.408Z] === RUN   TestNewCreateReplicationGroupInput/UnsetFieldsAreNilNotZeroType

[2020-08-05T01:42:19.408Z]     TestNewCreateReplicationGroupInput/UnsetFieldsAreNilNotZeroType: elasticache_test.go:220: NewCreateReplicationGroupInput(...): -want, +got:

[2020-08-05T01:42:19.408Z]           &elasticache.CreateReplicationGroupInput{

[2020-08-05T01:42:19.408Z]           	... // 5 identical fields

[2020-08-05T01:42:19.408Z]           	CacheParameterGroupName: nil,

[2020-08-05T01:42:19.408Z]           	CacheSecurityGroupNames: nil,

[2020-08-05T01:42:19.408Z]         - 	CacheSubnetGroupName:    nil,

[2020-08-05T01:42:19.408Z]         + 	CacheSubnetGroupName:    &"",

[2020-08-05T01:42:19.408Z]           	Engine:                  &"redis",

[2020-08-05T01:42:19.408Z]           	EngineVersion:           nil,

[2020-08-05T01:42:19.408Z]           	... // 19 identical fields

[2020-08-05T01:42:19.408Z]           }

[2020-08-05T01:42:19.408Z] --- FAIL: TestNewCreateReplicationGroupInput (0.00s)

[2020-08-05T01:42:19.408Z]     --- PASS: TestNewCreateReplicationGroupInput/AllPossibleFields (0.00s)

[2020-08-05T01:42:19.408Z]     --- FAIL: TestNewCreateReplicationGroupInput/UnsetFieldsAreNilNotZeroType (0.00s)
```